backend/db/migrations/20210717101512-addPolicies.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async () => {
+      await queryInterface.createTable('policies', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        shop_id: {
+          type: Sequelize.INTEGER
+        },
+        all_policies: {
+          type: Sequelize.JSON
+        },
+        auth_token: {
```

Comment:
I don't think we need the auth_token here. the shop_id should be sufficient.
The routes middleware will handle looking up the shop basd on the auth_token send by the front-end client.